niri-config/src/lib.rs

```diff
@@ -144,6 +144,8 @@ pub struct Xkb {
     pub options: Option<String>,
     #[knuffel(child, unwrap(argument))]
     pub file: Option<String>,
+    #[knuffel(child)]
+    pub numlock: bool,
```

Comment:
Does this need to be configurable? I'd just enable it always by default

Reply:
I'd argue yes.
I have a notebook laying around that overlays the numlock on top of regular keys.
Numlock has to be disabled by default on that machine for the keyboard to be usable.

Reply:
Oh god. Okay then, let's default it to enabled though

Reply:
I think defaulting to disabled would be better, since that matches the previous behaviour, and the way other things default to having numlock off too. I also used to have a laptop that had the numpad overlayed on top of the regular keys like that, too, so I imagine it's not too rare.

Reply:
I suggested just having it on in the default config. Similar to touchpad tap, etc